### DISCUSSION TITLE
Membatasi pemilihan varSelectInput?

### DISCUSSION THREAD

**Discussion by AGelfP** [teamalgoritma/community#576] [Q&A] [answered]
Saya ingin membuat user bisa melihat korelasi pergerakan antara dua saham. Data frame terlihat sebagai berikut:
![image](https://user-images.githubusercontent.com/122977451/221347487-54e0c31b-e6a4-4cf1-81b6-eda5973b5309.png)

Saya ingin menggunakan varSelectInput supaya pengguna bisa memilih dua saham yang akan dibandingkan. Melihat dari cheat sheet, saya lihat varSelectInput bisa dibuat supaya user dapat memilih lebih dari 1 kolom data frame. Namun apakah bisa supaya user hanya dapat memilih dua saja (tidak lebih dan tidak kurang)? So far, kode saya baru sampai sini, soalnya tidak ketemu cara membatasinya:
![image](https://user-images.githubusercontent.com/122977451/221347590-3c4227d0-6917-4f39-9104-a9ffb0e8c082.png)

Mohon bantuannya, terima kasih

EDIT: Pada saat ini solusi saya adalah untuk membuat dua bagian varSelectInput dan untungnya output muncul sesuai harapan saya:
![image](https://user-images.githubusercontent.com/122977451/221355691-157adde3-3e1b-4d9b-a093-78f3c19b7af2.png)

Namun bagi yang memiliki solusi yang mungkin lebih efisien, boleh di share yaa. Terima kasih

**Comment by idoali** [comment, marked as answer]
Terima kasih pak atas pertanyaannya.

Sebenarnya solusi yang bapak buat itu lebih tepat pak. Alasannya adalah ketika dashboard menampilkan dua input, user akan secara otomatis tau bahwa dia harus memasukkan dua input, tidak kurang dan tidak lebih. Bapak sebagai pembuat dashboard, tidak perlu susah-susah menjelaskan kepada user apa yang harus mereka lakukan. 

Untuk fungsi `varSelectInput()` sendiri kami tidak menemukan parameter yang bisa membatasi jumlah input selain `multiple=`. Dan seperti yang bapak tau sendiri, parameter ini hanya digunakan untuk memberi tahu apakah user bisa memberikan input lebih dari satu atau tidak.
